apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V37__preadj_rda_0_5_table_updates.sql

```diff
@@ -0,0 +1,170 @@
+ALTER TABLE "pre_adj"."FissClaims" ADD "medicalRecordNo" varchar(17);
+
+/*
+ * McsClaims
+ */
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrAssignment" varchar(1);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrClmLevelInd" varchar(1);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrHdrAudit" int;
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrHdrAuditInd" varchar(1);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrUSplitReason" varchar(1);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrJReferringProvNpi" varchar(10);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrJFacProvNpi" varchar(10);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrUDemoProvNpi" varchar(10);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrUSuperNpi" varchar(10);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrUFcadjBilNpi" varchar(10);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrAmbPickupAddresLine1" varchar(25);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrAmbPickupAddresLine2" varchar(20);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrAmbPickupCity" varchar(20);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrAmbPickupState" varchar(2);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrAmbPickupZipcode" varchar(9);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrAmbDropoffName" varchar(24);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrAmbDropoffAddrLine1" varchar(25);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrAmbDropoffAddrLine2" varchar(20);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrAmbDropoffCity" varchar(20);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrAmbDropoffState" varchar(2);
+ALTER TABLE "pre_adj"."McsClaims" ADD "idrAmbDropoffZipcode" varchar(9);
+
+/*
+ * McsDetails
+ */
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrTos" varchar(1);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrTwoDigitPos" varchar(2);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrDtlRendType" varchar(2);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrDtlRendSpec" varchar(2);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrDtlRendNpi" varchar(10);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrDtlRendProv" varchar(10);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrKDtlFacProvNpi" varchar(10);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrDtlAmbPickupAddres1" varchar(25);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrDtlAmbPickupAddres2" varchar(20);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrDtlAmbPickupCity" varchar(20);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrDtlAmbPickupState" varchar(2);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrDtlAmbPickupZipcode" varchar(9);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrDtlAmbDropoffName" varchar(24);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrDtlAmbDropoffAddrL1" varchar(25);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrDtlAmbDropoffAddrL2" varchar(20);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrDtlAmbDropoffCity" varchar(20);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrDtlAmbDropoffState" varchar(2);
+ALTER TABLE "pre_adj"."McsDetails" ADD "idrDtlAmbDropoffZipcode" varchar(9);
```

Comment:
DO we need an alter statement to update the MCS diagnosis code table `idr_diag_code` value to be not null?

Reply:
Thanks.  Will add that.